login.html

```diff
@@ -4,8 +4,44 @@
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>판다마켓</title>
+    <link rel="stylesheet" href="css/reset.css" />
+    <link rel="stylesheet" href="css/basic.css" />
+    <link rel="stylesheet" href="css/form.css" />
+    <link
+      rel="stylesheet"
+      as="style"
+      crossorigin
+      href="https://cdn.jsdelivr.net/gh/orioncactus/pretendard@v1.3.9/dist/web/static/pretendard.min.css"
+    />
   </head>
   <body>
-    LOGIN
+    <div class="form_container">
+      <a href="index.html">
+        <img
+          class="form_logo_img"
+          src="img/login_logo.png"
+          alt="로그인페이지_판다마켓_로고_이미지"
```

Comment:
💊 제안  
img 태그의 alt 속성은 alternative 라는 의미로, 이미지 파일을 다운로드하는 것에 실패해서 이미지 파일을 보여줄 수 없을 때  
어떤 이미지인지 파악할 수 있게 대신 제공되거나, 스크린리더로 읽혀지는 문자를 의미합니다.  
따라서 해당 이미지를 잘 표현할 수 있는 문자열을 넣어주면 됩니다.

```suggestion
          alt="판다마켓"
```